intake/gui/catalog/add.py

```diff
@@ -144,7 +144,7 @@ def move_down(self, *events):
                 fn = event.new[0]
                 if fn.endswith('/'):
                     if self.path_text.value:
-                        self.path_text.value = self.path_text.value + '/' + fn
+                        self.path_text.value = os.path.join(self.path_text.value, fn)
```

Comment:
This will fail on Windows, where the separator is "\\". Perhaps `posixpath.join`?

Reply:
@martindurant, isn't this the situation?  This is what I get on Windows:
```python
a = 's3://bucket/key1/'
b = 'key2/'
print(os.path.join(a,b))

's3://bucket/key1/key2/'
```

Reply:
What if `a = 'bucket'` ?

Reply:
Can that be the case?  I thought the GUI puts a "/" on the end.

Reply:
I suppose you are right, if the path ends with "/", it amounts to the same thing. This is a surprise to me! I never work with windows paths for a reason.